docs/release-notes/release-notes-0.15.0.md

```diff
@@ -44,6 +44,12 @@ then watch it on chain. Taproot script spends are also supported through the
 * Add [--socksproxy](https://github.com/lightningnetwork/lnd/pull/6422)
   to allow for RPC calls via Tor.
 
+## Neutrino
+
+[Neutrino now suports BIP
+155](https://github.com/lightningnetwork/lnd/pull/6468), allowing it to connect
+to Bitcoin nodes that advertise a Tor v3 onion service address.
```

Comment:
It could previously connect, just couldn't gossip

Reply:
@Crypt-iQ No, it's currently not possible for Neutrino to connect to a v3 service.

The error message is
```
[WRN] BTCN: unable to lookup IP for 0230a5bca558e6741460c13dd34e636da28e52afd91cf93db87ed1b0392a7466eb@nwozjjgxilimzjd3d23vfgkmdsr6j2wekia4d77f673cefrfot6wtqqd.onion:9735: tor general error
```